package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "airtable",
-  "version": "0.12.2",
+  "name": "bernatfortet.airtable",
```

Comment:
Believe you will want to remove `bernatfortet` from this

Also is there any reason the version has been bumped to 0.12.5 instead of 0.12.3?